srcCpp/perftest_publisher.cxx

```diff
@@ -518,6 +542,11 @@ bool perftest_cpp::ParseConfig(int argc, char *argv[])
                 return false;
             }
             _NumSubscribers = strtol(argv[i], NULL, 10);
+            _MessagingSocketsArgv[_MessagingSocketsArgc] =
+                    DDS_String_dup(argv[i - 1]);
+            _MessagingSocketsArgv[++_MessagingSocketsArgc] =
```

Comment:
You can use _MessagingSocketsArgc++ in the previous string and just do _MessagingSocketsArgc and not ++_MessagingSocketsArgc